src/jquery.ime.js

```diff
@@ -1,17 +1,41 @@
 ( function ( $ ) {
 	'use strict';
+	var TextEntryFactory, TextEntry, FormWidgetEntry, ContentEditableEntry,
+		defaultInputMethod;
 
 	// rangy is defined in the rangy library
 	/*global rangy */
 
 	/**
+	 * Minimal functionality from OOjs: Initialize a class for OO inheritance
```

Comment:
I think it's worth mentioning that this is based on OOJS so in future we don't add code which may conflict with it, and therefore leave the door open to actually using it.